src/hooks/useNow.ts

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { getMillisecondsToTomorrow } from "../lib/date";
+
+/**
+ * Обновляемая дата сегодняшнего дня
+ *
+ * Дата - сегодня (в соответствии с системным временем)
+ *
+ * Часы, минуты, секунды, миллисекунды - произвольные
+ */
+export function useNow() {
+  const [now, setNow] = React.useState(new Date());
```

Comment:
```suggestion
  const [now, setNow] = React.useState(() => new Date());
```
давай обернём в функцию, чтобы не создавать в пустую объект при ре-рендере